models/stics/README.md

```diff
@@ -0,0 +1,64 @@
+A generic template for adding a new model to PEcAn
```

Comment:
This looks like it's still fully generic -- does it need STICS-specific information now or is that waiting for a later PR?